app/src/main/java/blockeq/com/stellarwallet/helpers/BottomNavigationShiftMode.kt

```diff
@@ -0,0 +1,28 @@
+package blockeq.com.stellarwallet.helpers
+
+import android.annotation.SuppressLint
+import android.support.design.internal.BottomNavigationItemView
+import android.support.design.internal.BottomNavigationMenuView
+import android.support.design.widget.BottomNavigationView
+import android.util.Log
+import blockeq.com.stellarwallet.helpers.Constants.Companion.TAG
+
+@SuppressLint("RestrictedApi")
+fun BottomNavigationView.disableShiftMode() {
+    val menuView = getChildAt(0) as BottomNavigationMenuView
+    try {
+        val shiftingMode = menuView::class.java.getDeclaredField("mShiftingMode")
+        shiftingMode.isAccessible = true
+        shiftingMode.setBoolean(menuView, false)
+        shiftingMode.isAccessible = false
+        for (i in 0 until menuView.childCount) {
+            val item = menuView.getChildAt(i) as BottomNavigationItemView
```

Comment:
Would it be possible to apply our custom Font here? Since we are already retrieving child view, we can get the Text and apply the font...see https://stackoverflow.com/questions/45144314/bottomnavigationview-how-to-change-menu-item-font.